framework/include/security/security_api.h

```diff
@@ -23,4 +23,9 @@
 #include "security_keymgr.h"
 #include "security_ss.h"
 
+#if (defined(CONFIG_ARCH_BOARD_ARTIK053S) || defined(CONFIG_ARCH_BOARD_ARTIK053)) \
+			&& defined(CONFIG_S5J_HAVE_SSS)
+#include "../../../os/arch/arm/src/s5j/sss/isp_custom.h"
```

Comment:
Could you let me know why this board specific header file is required in this framework?

Reply:
In future, It will be removed.
But currently, It needs.